lib/Sema/ConstraintSystem.cpp

```diff
@@ -2814,6 +2814,9 @@ static void diagnoseOperatorAmbiguity(ConstraintSystem &cs,
                     operatorName.str());
         return;
       }
+    } else if (operatorName.is("~=")) {
```

Comment:
It might be better to compare with `Id_MatchOperator`

Reply:
Done!